dbus/methods_test.go

```diff
@@ -739,9 +754,527 @@ func TestStartStopTransientUnit(t *testing.T) {
 	}
 }
 
+// Ensure that oneshot transient unit starting and stopping works.
+func TestStartStopTransientUnitOneshot(t *testing.T) {
```

Comment:
@jonboulle I rewrote the 2nd patch, and force-pushed. With a test table, it will be more readable than before. PTAL.